dataactvalidator/scripts/loadFSRS.py

```diff
@@ -28,6 +29,12 @@ def maxCurrentId():
 
 
 if __name__ == '__main__':
-    num_awards, num_subawards = loadFSRS(maxCurrentId() + 1)
-    print("Inserted/Updated {} awards, {} subawards".format(
-        num_awards, num_subawards))
+    if not configValid():
+        print("No config key for broker/fsrs_service/wsdl")
```

Comment:
Thanks for adding a check here! Is this a place where we'd want to use your suggested logging method?

That shouldn't keep us from proceeding--more just wondering if the general pratice is to do logging wherever you're displaying this type of info or if it's more nuanced than that.